packages/uniforms-antd/src/AutoFields.tsx

```diff
@@ -24,6 +26,12 @@ export default function AutoFields({
     props,
     (fields ?? schema.getSubfields())
       .filter(field => !omitFields.includes(field))
-      .map(field => createElement(autoField, { key: field, name: field })),
+      .map(field =>
+        createElement(autoField, {
+          key: field,
+          name: field,
+          ...(showInlineError !== null ? { showInlineError } : {}),
+        }),
```

Comment:
Two things here. One, it should be `undefined`, not `null`. Two, we can get rid of one temporary object. My suggestion:
```suggestion
        createElement(
          autoField,
          Object.assign(
            { key: field, name: field },
            showInlineError === undefined ? null : { showInlineError },
          ),
        ),
```